ToDo-Garden/ToDo-Garden/SettingScene/Sources/SettingScene/Views/SettingCollectionViewCell.swift

```diff
@@ -0,0 +1,210 @@
+  }
+
+  private func addBottomBorderLayer() -> CALayer {
+    let bottomLayer = CALayer()
+    bottomLayer.frame = CGRect(
+      x: self.bounds.minX,
+      y: self.bounds.maxY - 1.0,
+      width: self.bounds.width,
+      height: 1.0
+    )
+    bottomLayer.name = SubLayerName.trailing
+    return bottomLayer
+  }
+
+  private func addTrailingBorderLayer() -> CALayer {
+    let trailingLayer = CALayer()
+    trailingLayer.frame = CGRect(
+      x: self.bounds.maxX - 1.0,
+      y: self.bounds.minY,
+      width: 1.0,
+      height: self.bounds.height
+    )
+    trailingLayer.name = SubLayerName.trailing
+    return trailingLayer
+  }
+
+  private func removeSubLayersInMiddle() {
+    self.layer.sublayers?.forEach { (subLayer: CALayer) in
+      if subLayer.name == SubLayerName.leading ||
+        subLayer.name == SubLayerName.bottom ||
+        subLayer.name == SubLayerName.trailing ||
+        subLayer.name == CALayer.SubLayerName.bottomRoundedBorder {
+        subLayer.removeFromSuperlayer()
+      }
+    }
+  }
+}
+
+// MARK: Private Functions
+
+extension SettingCollectionViewCell {
+  private func setup() {
+    self.backgroundColor = UIColor.toDoGardenWhite
+    self.setupRightForwardImageView()
+    self.setupTitleLabel()
+  }
+
+  private func setupRightForwardImageView() {
+    self.rightForwardImageView.image = UIImage.forwardButtonImage
+    self.setupRightForwardImageViewLayout()
+  }
+
+  private func setupTitleLabel() {
+    self.titleLabel.font = UIFont.pretendardBodyRegular
+    self.titleLabel.textColor = SettingSceneTheme.mainColor
+    self.setupTitleLabelLayout()
+  }
+}
+
+extension SettingCollectionViewCell {
+  private func setupRightForwardImageViewLayout() {
+    self.contentView.addSubview(self.rightForwardImageView)
+    self.rightForwardImageView.usingAutolayout()
+
+    NSLayoutConstraint.activate(
+      [
+        self.rightForwardImageView.centerYAnchor.constraint(equalTo: self.contentView.centerYAnchor),
+        self.rightForwardImageView.trailingAnchor.constraint(
+          equalTo: self.contentView.trailingAnchor,
+          constant: -8
+        ),
+        self.rightForwardImageView.widthAnchor.constraint(equalToConstant: 24),
+        self.rightForwardImageView.heightAnchor.constraint(equalToConstant: 24)
+      ]
+    )
+  }
+
+  private func setupTitleLabelLayout() {
+    self.contentView.addSubview(self.titleLabel)
+    self.titleLabel.usingAutolayout()
+
+    NSLayoutConstraint.activate(
+      [
+        self.titleLabel.centerYAnchor.constraint(equalTo: self.contentView.centerYAnchor),
+        self.titleLabel.leadingAnchor.constraint(
+          equalTo: self.contentView.leadingAnchor,
+          constant: 8
+        ),
+        self.titleLabel.trailingAnchor.constraint(equalTo: self.rightForwardImageView.leadingAnchor)
+      ]
+    )
+  }
+}
+
+extension SettingCollectionViewCell {
+  enum SubLayerName: CaseIterable {
+    static let leading = "leading"
+    static let trailing = "leading"
```

Comment:
@wood0203 
`p1`
trailing으로 변경되어야할 것 같아요~!

```suggestion
    static let trailing = "trailing"
```